deployment/base/rbac-master-topologyupdater/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: node-feature-discovery
+
+resources:
+- master-topologyupdater-serviceaccount.yaml
+- master-topologyupdater-clusterrole.yaml
+- master-topologyupdater-clusterrolebinding.yaml
```

Comment:
I think you could drop this "base" altogether

Reply:
Ack